episodes/transfer_data.md

```diff
@@ -29,13 +29,71 @@ Objectives
 
 ## Exercises
 
-::::{dropdown} Challenge Title
+::::{dropdown} What should you store in your home directory?
 :open:
 
-Challenge content
+Select all that apply:
+
+- [ ] Notebooks
+- [ ] Code
+- [ ] Large and/or shared datasets with read-write permissions
+- [ ] Text files
+- [ ] Small datasets
+- [ ] Configuration files
+- [ ] Read-only datasets, e.g for teaching
+- [ ] Temporary data, e.g. intermediate results from a calculation
+
+:::{dropdown} Solution
+
+- [x] Notebooks
+- [x] Code
+- [ ] Large and/or shared datasets with read-write permissions – *consider using Cloud Object Storage or other cloud-native storage solutions*
+- [x] Text files
+- [x] Small datasets
+- [x] Configuration files
+- [ ] Read-only datasets, e.g for teaching – *consider placing these files in the `~/shared` directory*
+- [ ] Temporary data, e.g. intermediate results from a calculation – *consider using the `/tmp` directory*
+
+:::
+
+::::
+
+:::{dropdown} Data requirements
+:open:
+
+Think about the data that you or your community require for your/their work.
+
+- What type of data is it? E.g. tabular data (.csv, .xls), images (.png, .jpg, .bmp), other scientific data formats (.h5, .nc, .fastq)
+- How large is the dataset?
+- Where is the data stored?
+- Is the same dataset used by a group of collaborators?
+
+Write up your thoughts in our shared collaborative document.
```

Comment:
Will we have a collaborative notes document when working through this episode? Is the plan for us to have a rolling notes doc, which people can contribute to when following the lesson on their own?

Reply:
Yes I will use a collaborative notes document for the in-person sessions and I think it will be good to use it for the self-study portion too for a sense of continuity throughout the entire workshop 📓 

I'll make sure to record this in the lesson setup pages.